cumulus_library/studies/core/observation.sql

```diff
@@ -0,0 +1,40 @@
+drop table if exists core__observation;
```

Comment:
`drop` statements are no longer required to be explicitly defined - all tables/views beginning with the study prefix will be discovered directly from the DB and auto dropped.